packages/connect-examples/webextension-v3/manifest.json

```diff
@@ -0,0 +1,14 @@
+{
+    "manifest_version": 3,
+    "name": "Trezor Suite",
```

Comment:
nitpick: I'd change this to something like
```
"name": "@trezor/connect webextension mv3 example",
```
is it too long? :thinking: 
and maybe unify it with mv2 example folder as well

Reply:
Updated https://github.com/trezor/trezor-suite/pull/7699/commits/420dd87e3e07271febe4be829f7b3ecfbf313c37